specification/paths/Payments-v1-providers.json

```
@@ -0,0 +1,84 @@
+{
+  "get": {
+    "tags": [
+      "PaymentProviders"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "organizations.manage"
+        ]
+      }
+    ],
+    "summary": "Get all payment providers.",
+    "description": "This endpoint retrieves payment providers available to your API client.",
+    "parameters": [
+      {
+        "name": "shop_id",
+        "in": "query",
+        "description": "When specified only payment providers connected to the given shop are returned.",
+        "schema": {
+          "type": "string"
+        }
```

Let's use the existing shop filter, which we also use on our `/returns/v1/returns` endpoint.

```suggestion
        "$ref": "#/components/parameters/query-filter-shop"
```

---

It is not really a filter though, I prefer we stick with `shop_id` query parameter because the intended behavior is different that the filter